src/Eloquent/HasDocumentVersion.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Eloquent;
+
+trait HasDocumentVersion
```

Comment:
"Document Version" sounds like we are versioning the document with revisions, with an history. Naming everything "Schema Version" would be more appropriate.

Reply:
I agree with you, I will make the changes